packages/connectivity_plus/connectivity_plus/android/src/main/java/dev/fluttercommunity/plus/connectivity/ConnectivityBroadcastReceiver.java

```diff
@@ -78,12 +78,12 @@ public void onCancel(Object arguments) {
   @Override
   public void onReceive(Context context, Intent intent) {
     if (events != null) {
-      events.success(connectivity.getNetworkType());
+      events.success(String.join(",", connectivity.getNetworkTypes()));
```

Comment:
`success()` accepts an `Object`, so I think it would be possible to pass an array here, instead of a String. I'd prefer if we can keep the data as it is, rather than serializing/deserializing, if not strictly necessary. Would that be possible?

Reply:
Agreed. I refactored the code to support the exchange list type between flutter and a host system in all integrations.